src/components/individuals/IndividualsListContent.js

```diff
@@ -61,7 +61,7 @@ const IndividualsListContent = ({
                 columns={TABLE_COLUMNS}
                 items={individuals}
                 itemsByID={individualsByID}
-                rowKey="id"
+                rowKey="label"
```

Comment:
Is label appropriate here? Is label always unique? Isn't `rowKey` used with `itemsByID`?

Reply:
Label is always unique but yeah i will double check if it is used with itemsByID (i guess you already checked)